data/darktableconfig.xml.in

```diff
@@ -447,6 +447,13 @@
     <shortdescription>jobcode of import job</shortdescription>
     <longdescription/>
   </dtconfig>
+  <dtconfig prefs="gui">
+    <name>plugins/collect/descending</name>
+    <type>bool</type>
+    <default>true</default>
+    <shortdescription>sort collection recent to early</shortdescription>
```

Comment:
I would have said : "recent to older" but if you are a native English speaking forget about what I said :)

Reply:
I am, but I've been too long in France :-) I'm good with older.